misc/python/materialize/mzcompose/composition.py

```diff
@@ -710,7 +710,7 @@ def up(
             *(["--detach"] if detach else []),
             *(["--wait"] if wait else []),
             *services,
-            max_tries=2,
+            max_tries=max_tries,
```

Comment:
Oh, that's embarassing

Reply:
It happens, I'm more surprised that our Python linting didn't catch it.